apypie/api.py

```diff
@@ -189,6 +189,8 @@ def http_call(self, http_method, path, params=None, headers=None, options=None,
 
         request = self._session.request(http_method, full_path, **kwargs)
         request.raise_for_status()
+        if request.status_code == 204:  # no content
+            return None
```

Comment:
this is not an empty dict ;) (and I know you changed that after opening the PR)
what was the reasoning behind the change?

I'm fine either way, just want to understand the difference

Reply:
oh, there was previous discussion on that. all good.